docs/docs/conceptual/choosing-a-cms.md

```diff
@@ -0,0 +1,69 @@
+
+- **one specialized CMS** (Shopify).
+
+**Second**, consider the price point:
+
+- If you're working on a personal project or prototype, a few of these have a generous free tier (Contentful, Sanity, DatoCMS, Strapi, Prismic).
+
+- If you're looking more at "Team", "Pro", or "Business" price points (eg $29, $99, or \$299 per month), the above CMSs are all good options, plus Wordpress and Drupal .
+
+- If you're looking more at an "enterprise" project in the four digits per month and up, Contentful is the leader, with others using Contentstack, Sanity, and Strapi.
+
+**Third,** consider project requirements.
+
+**Contentful** is the most common CMS used with Gatsby, accounting for roughly half of all CMS usage with Gatsby. It's the most mature headless CMS as measured by revenue, number of customers, and venture funding. In some ways, it's become the "default" to use with Gatsby.
+
+Users choosing other CMSs typically have a specific reason for their choice. While every user has different reasons for their choice, some trends we see users choosing **Sanity** or **Strapi** for the developer-friendliness or if they need something on-premise; **DatoCMS** if they will run into Contentful's model limits and want a lower-priced alternative, **Prismic** if they like the content editing UI; **Contentstack** if they like the editing UI and have an enterprise budget; **Wordpress** when the client or content team is already familiar with the Wordpress UI. **Drupal** if open-source, configurability / custom code are important.
+
+## Using multiple CMS systems together
+
+Another approach is, rather than using one CMS, use different CMSs for different parts of the website. Known as a ["content mesh" approach](https://www.gatsbyjs.com/blog/2018-10-04-journey-to-the-content-mesh/), the typical rationale here is that one CMS provides must-have specialized functionality but is less good for the rest of the site. Typically the decision is to use a specialized CMS for part of the website, and a general-purpose CMS for most of the website.
+
+Typically there are two reasons to intentionally take this approach:
+
+### Using Shopify as an e-commerce system
+
+Shopify has best-in-class e-commerce functionality, but the rest of their interface (for example, their blogs feature) is often seen as subpar compared to other systems. As a result, it's very standard to use Shopify as the backend for the store, but a general purpose CMS like Contentful as the backend for the rest of the site.
+
+### Using Wordpress as a blogging tool
+
+Because Wordpress is so familiar to content authors, and has a best-in-class content composition experience, it's quite common to move the blog portion of the website with Gatsby, and use a flexible content modelling CMS for the rest of the website.
+
+Apollo does this [for their blog](https://twitter.com/apollographql/status/1250479066605662210), as we do for the blog at [gatsbyjs.com/blog](gatsbyjs.com/blog).
+
```

Comment:
Restructuring this to try and make it easier to scan. Lmk what you think.

```suggestion
Can't choose just one CMS? You can use different CMSs for different parts of the website. Known as a ["content mesh" approach](https://www.gatsbyjs.com/blog/2018-10-04-journey-to-the-content-mesh/), the typical rationale here is that one CMS provides must-have specialized functionality but is less good for the rest of the site.

Typically, teams that use multiple CMSs use a specialized CMS for part of the website and a general-purpose CMS for the rest of the website. There are two common reasons teams choose this approach:

- **Using Shopify as an e-commerce system.**
    - Shopify has best-in-class e-commerce functionality, but the rest of their interface (for example, their blogs feature) is often seen as subpar compared to other systems. As a result, it's very standard to use Shopify as the backend for the store, with a general-purpose CMS like Contentful as the backend for the rest of the site.
- **Using WordPress as a blogging tool.**
    - Because WordPress is so familiar to content authors and has a best-in-class content composition experience, it's quite common to move the blog portion of the website with Gatsby, and use a flexible content modelling CMS for the rest of the website. (Apollo does this (source: [tweet from Apollo](https://twitter.com/apollographql/status/1250479066605662210)), as does the [Gatsby blog](gatsbyjs.com/blog).)

```